examples/basic/basic_pysdk_demo_image.ipynb

```diff
@@ -0,0 +1,146 @@
+    "2. Run inference on a DeGirum AI Server deployed on the local host or on some computer in your LAN or VPN;\n",
+    "3. Run inference on a DeGirum ORCA accelerator directly installed on your computer.\n",
+    "\n",
+    "To try different options, you need to specify the appropriate `hw_location` option. \n",
+    "\n",
+    "You also need to specify your cloud API access token in `degirum_cloud_token`.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17df0fd4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# make sure degirum-tools package is installed\n",
+    "!pip show degirum-tools || pip install degirum-tools"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "979779d6",
+   "metadata": {},
+   "source": [
+    "#### Specify where you want to run your inferences, model zoo url, model name and image source"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "313e14a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# hw_location: where you want to run inference\n",
+    "#     \"@cloud\" to use DeGirum cloud\n",
+    "#     \"@local\" to run on local machine\n",
+    "#     IP address for AI server inference\n",
+    "# model_zoo_url: url/path for model zoo\n",
+    "#     cloud_zoo_url: valid for @cloud, @local, and ai server inference options\n",
+    "#     '': ai server serving models from local folder\n",
+    "#     path to json file: single model zoo in case of @local inference\n",
+    "# model_name: name of the model for running AI inference\n",
+    "# image_source: image source for inference\n",
+    "#     path to image file\n",
+    "#     URL of image\n",
+    "#     PIL image object\n",
+    "#     numpy array\n",
+    "# degirum_cloud_token: your token for accessing the DeGirum cloud platform\n",
+    "hw_location = \"@cloud\"\n",
+    "model_zoo_url = \"https://cs.degirum.com/degirum/public\"\n",
+    "model_name = \"mobilenet_v2_ssd_coco--300x300_quant_n2x_orca1_1\"\n",
+    "image_source = \"https://raw.githubusercontent.com/DeGirum/PySDKExamples/main/images/TwoCats.jpg\"\n",
+    "degirum_cloud_token = \"<enter your token here>\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df6dfebd",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "#### The rest of the cells below should run without any modifications"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fea1e8c8",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import degirum as dg\n",
+    "\n",
+    "# Connect to AI inference engine\n",
+    "# hw_location: hardware location\n",
+    "# model_zoo_url: cloud model zoo URL\n",
+    "# degirum_cloud_token: Degirum cloud API access token\n",
+    "zoo = dg.connect(hw_location, model_zoo_url, degirum_cloud_token)\n",
+    "\n",
+    "# Load object detection AI model, setting some of the model properties\n",
```

Comment:
Would be good to add the zoo.list_models() line here, and explain that any of the strings returned by it can be used as model_name in the subsequent zoo.load_model() call. Unless the goal is to create a truly minimal example of 4 lines, that information is always useful.

Reply:
@AlexABolotov added another cell with the `zoo.list_models()` command to explain